### DISCUSSION TITLE
VSCode extension

### DISCUSSION THREAD

**Discussion by nifty-coder** [community/community#61763] [Programming Help]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Hey developers. I made a vscode extension : https://marketplace.visualstudio.com/items?itemName=nifty-coder.simply-voice-code

In my readme, i mention this for USAGE: 
1.Open a coding file in Visual Studio Code.
2.Press Ctrl/Cmd + Shift + P to open the command palette.
3.Search for "Start Voice Coding" and select it to initiate voice coding.
4.Allow microphone access when prompted.
5.Speak a prompt to generate code. For example, say "Create a function that adds two numbers."
6.The generated code will be displayed with a confirmation prompt. You can choose to insert the code or cancel.

The problem is when I install it, I'm not able to see "Start Voice Coding" when I open the command palette.

Please help me!!!  Github repo: https://github.com/nifty-coder/vscode-simply-voice-code-extension

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @nifty-coder ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
